### DISCUSSION TITLE
Theme-aware images?

### DISCUSSION THREAD

**Discussion by paxbun** [icerockdev/moko-resources#479] [Q&A]
Does moko-resources support making light and dark modes versions of the same image resource? It seems moko-resources only supports light/dark color value support... I'm manually checking `isSystemInDarkTheme` like the following.
```
painterResource(
    if (isSystemInDarkTheme()) {
        MR.images.logoDark
    } else {
        MR.images.logo
    }
)
```

Edit:

It seems moko-resources also does not support image localization. I'm using Compose Multiplatform, and using Java's `Locale` should not be possible when targeting iOS. what should I do in this case?

**Comment by Alex009** [comment]
it's available in 0.24.0
